packages/tree/src/notebook-tree.ts

```diff
@@ -19,4 +19,11 @@ export class NotebookTreeWidget extends TabPanel implements INotebookTree {
     });
     this.addClass('jp-TreePanel');
   }
+
+  addWidget(widget: Widget, activate = true): void {
```

Comment:
No, it can be let to the plugin. I added it because I was thinking that creating a new tab was to activate it (except for the initial ones). I'll remove it.